From 5983afa22e9b571ab83882e38c0588566257e5fe Mon Sep 17 00:00:00 2001
From: Li Zhou <li.zhou@windriver.com>
Date: Fri, 19 Aug 2022 10:08:12 +0800
Subject: [PATCH] efitools: prepare keys

Copy uefi keys (example keys) to the proper path for building.
Replace the DB.crt (example key) with tis-boot.crt (public key
in use for verifying signed shim image).

Signed-off-by: Li Zhou <li.zhou@windriver.com>
Signed-off-by: Virgil Bucoci <virgil.bucoci@windriver.com>
---
 debian/rules | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/debian/rules b/debian/rules
index 89115b3..0c44369 100755
--- a/debian/rules
+++ b/debian/rules
@@ -4,6 +4,11 @@
 # Uncomment this to turn on verbose mode.
 export DH_VERBOSE=1
 
+override_dh_auto_build:
+	cp debian/uefi_sb_keys/* ./
+	mv tis-boot.crt DB.crt
+	dh_auto_build
+
 override_dh_auto_install:
 	dh_auto_install -- EFIDIR="debian/efitools/usr/lib/efitools/${DEB_TARGET_MULTIARCH}"
 
-- 
2.47.3

